pkg/networkservice/common/mechanisms/kernel/server.go

```diff
@@ -33,12 +32,12 @@ type mechanismsServer struct{}
 
 func (m mechanismsServer) Request(ctx context.Context, req *networkservice.NetworkServiceRequest) (*networkservice.Connection, error) {
 	conn := req.Connection
-	inode := uint64(0)
+	inode := uintptr(0)
```

Comment:
Why uintptr here?

Reply:
It's guaranteed to hold a pointer based on the platform. This will be important if we see 32 bit platforms. E.g. most Raspberry PIs are 32 bit.